### PR TITLE
Fixing kerning para style

### PR DESCRIPTION
**Pull request by vikas-chaudhary-2802**
### Description

I have added the kerning para style --> #520 
This PR introduces a new feature to the `Scarpe` library - a kerning para style. Kerning is an important aspect of typography that adjusts the spacing between characters to achieve a visually pleasing result. This feature will allow users to apply kerning to their paragraphs, enhancing the visual appeal of their text.

### Image(if needed, helps for a faster review)

Screenshot 2024-02-08 at 10 18 03 PM

### Checklist

- [Yes] Run tests locally

### CODE REVIEW

**Review by noahgibbs — APPROVED** [review]

**Comment by noahgibbs** [comment]
Looks good!

**Comment by noahgibbs** [comment]
Looks like you'll need to generate HTML fixtures, since you added a new example. You should be able to do that by running "rake test:regenerate_html_fixtures" and adding the generated file to this PR.

File: lacci/lib/shoes/drawables/para.rb [hunk 1]
```diff
@@ -3,7 +3,7 @@
 class Shoes
   class Para < Shoes::Drawable 
   include FontHelper
-    shoes_styles :text_items, :size, :family, :font_weight, :font, :font_variant, :emphasis
+    shoes_styles :text_items, :size, :family, :font_weight, :font, :font_variant, :emphasis, :kerning
```
Comment (Schwad) [inline lacci/lib/shoes/drawables/para.rb:6]: @noahgibbs when we get these new excellent features; should we add them to manual.md you think?

Unless I'm being silly and kerning was an existing thing in manual.md 😅

Comment (noahgibbs) [reply lacci/lib/shoes/drawables/para.rb:6]: Kerning is in there already! We just only had it *work* recently :-)

There's a surprising amount of Shoes functionality we're still adding.

**Pull request merged**